chef/cookbooks/crowbar/templates/default/crowbar-router-migration.sh.erb

```diff
@@ -35,17 +35,23 @@ set +x
 source /root/.openrc
 set -x
 
+if ssh $hostname "test -e $UPGRADEDIR/crowbar-router-migrated" ; then
+    log "l3 agent was already evacuated from host $hostname"
+    touch $UPGRADEDIR/crowbar-router-migration-ok
+    exit 0
+fi
```

Comment:
As said, it looks a little weird to do it this way here. But if other reviewers are ok with it am willing to +1 it as well.